src/CMakeLists.txt

```diff
@@ -23,9 +23,9 @@ rock_library(gazebo_underwater_msgs
 rock_library(gazebo_underwater
     SOURCES GazeboUnderwater.cpp
     HEADERS GazeboUnderwater.hpp
+    DEPS_TARGET Boost::system
```

Comment:
Where is this target being defined? Shouldn't you call `find_package(Boost REQUIRED COMPONENTS system)` first?

Reply:
It's already done at the top of the file.

Btw, updated because the FindBoost version on 18.04 does not use targets.